### PR TITLE
DS-782 square corners search button

### PR DESCRIPTION
**Pull request by kimlarocca**
DS-750 no focus border on search box or button 

(kim: i saw this was approved to deploy)

### CODE REVIEW

File: app/styles/_library/_objects.forms.scss [hunk 1]
```diff
@@ -55,12 +55,22 @@ div.o-confirm-message p {
 .c-search__button {
   height: 100%;
   padding: 10px;
+  margin: 0px;
+  border-radius: 0px;
+}
+
+.c-search__button:focus {
```
Comment (walsh9) [inline app/styles/_library/_objects.forms.scss:62]: We can't just remove the focus outline without providing an alternative focus style.  I've sent the ticket back to design for feedback.

Comment (walsh9) [reply app/styles/_library/_objects.forms.scss:62]: And it looks like this was already merged as #75 ? Not sure what to do with this now...

Comment (kimlarocca) [reply app/styles/_library/_objects.forms.scss:62]: Maybe it was, I’m not sure. I saw Brad was offline and that the ticket got approved and matched it to this open branch.

Comment (walsh9) [reply app/styles/_library/_objects.forms.scss:62]: guess i can just release with the bug for now to get things out and leave the pr and jira ticket open.

Comment (bkammin) [reply app/styles/_library/_objects.forms.scss:62]: late to the game here -- i'll follow up with design/product to resolve this. the focus border was being flagged as bug so i took it out.

**Comment by bkammin** [comment]
Closing this in favor of specific fix in #84

**Pull request closed without merging**